src/filetree.rs

```diff
@@ -294,7 +294,8 @@ pub(crate) fn apply_diff(
     for pathstr in diff.additions.iter().chain(diff.changes.iter()) {
         let path = Path::new(pathstr);
         if let Some(parent) = path.parent() {
-            // TODO: care about directory modes?  We don't for FAT.
+            // Note that for FAT there are no unix permission bits.
+            // See also https://github.com/coreos/fedora-coreos-config/commit/8863c2b34095a2ae5eae6fbbd121768a5f592091
```

Comment:
Minor: I was following the referenced PR/discussion and the world is settling on `0700`. I noted your `Perhaps we should be 0700 in all of these directories. (Though if the toplevel /boot/efi is 0700 then most of the other perms don't matter)` and I somehow agree that in order to avoid surprises/inconsistencies it may be good to align child directories on `0700`.
Did you forget to update the line below here, did you change your mind, or did I mix up something?

Reply:
Well it's more that it doens't matter really what we pick here.  But I updated to use `0700` here too.